customization-guide.md

```diff
@@ -0,0 +1,106 @@
+# IMS Provisioning for Wazi environment
+
+The IMS provisioning operator playbooks demonstrate how to allocate the required
+data sets and configure them to provision IMS and related services.
+
+It is a good practice to review the playbook contents before executing
+them. It will help you understand the requirements in terms of space, location,
+names, authority, and the artifacts that will be created and cleaned up.
+Although playbooks are written to operate without the need for the user’s
+configuration, flexibility is written into the playbooks because it is not easy
+to determine if a playbook has access to the host’s resources. Review the
+playbook notes sections for additional details and configuration.
+
+## Requirements
+* IBM z/OS core collection 1.5.0
```

Comment:
The z/OS Core collection requirement here does not match the `requirements.yml` which states `1.6.0`